lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -57,5 +57,10 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// Sass variables within a namespace (e.g. namespace.$var: x)
+	if (prop.includes('.$') && prop[0] !== '.' && prop[prop.length - 1] !== '$') {
```

Comment:
Let's just go with `if (prop.includes('.$'))` as the util is more interested in excluding non-standard properties than the validity of name-spaced dollar variables.